evalai/utils/submissions.py

```diff
@@ -0,0 +1,58 @@
+import requests
+import sys
+
+from beautifultable import BeautifulTable
+from click import echo, style
+
+from evalai.utils.auth import get_request_header
+from evalai.utils.common import validate_token
+from evalai.utils.urls import URLS
+from evalai.utils.config import API_HOST_URL, EVALAI_ERROR_CODES
+
+
+def pretty_print_submission_table(submissions):
+    """
+    Funcion to print the submissions for a particular Challenge.
+    """
+    table = BeautifulTable()
+    attributes = ["participant_team_name", "method_name", "execution_time", "status", "submitted_at"]
+    table.column_headers = attributes
+
+    for submission in submissions:
+        team_name = submission["participant_team_name"]
+        method_name = submission["method_name"]
+        execution_time = submission["execution_time"]
+        status = submission["status"]
+        submitted = submission["submitted_at"].split("T")[0]
+        value = [team_name, method_name, execution_time, status, submitted]
```

Comment:
Why not iterate through the `attributes` list and access the dict from the strings there. Then you can add a single line of code to update the `submitted` value with the split (and add a comment to indicate why). That should simplify this code tremendously.
Also, I assume that T in `submitted_at` is due to the UTC standard. You may want to consider keeping it in.

Reply:
@varunagrawal This is enough, right?